modules/mod_translation/controllers/controller_language_set.erl

```diff
@@ -40,6 +40,7 @@ resource_exists(Context) ->
 previously_existed(Context) ->
     {true, Context}.
 
+-spec moved_temporarily(#context{}) -> tuple().
```

Comment:
The spec in line 43 uses a record, please define a type for the record and use that instead.